app/src/main/java/it/quip/android/util/MockUtils.java

```diff
@@ -58,6 +70,10 @@ public static Circle circleWithNameAndMembers(String name, List<User> members) {
         return circle;
     }
 
+    public static Quip quipWithText(String text) {
+        return new Quip(randomId(), text, randomUser(), randomUser(), randomCircle(), randomTimestamp());
```

Comment:
I don't think we're going to use this constructor outside of MockUtils so we probably shouldn't create it. The reason the other methods like this are using JSON is to go through the public interfaces we've already define